frontend/public/locales/es/about.json

```diff
@@ -1,11 +1,11 @@
 {
-  "aboutTheClothingLoop": "Acerca de The Clothing Loop",
+  "aboutTheClothingLoop": "Acerca de the Clothing Loop",
   "p1": "<p><strong>The Clothing Loop es una iniciativa que ofrece una manera fácil de intercambiar ropa con otras personas en su propio vecindario.</strong> ¡Es divertido, gratis y sostenible! La idea es simple: bolsas (grandes) llenas de ropa que recorren una ruta que pasa por todos los participantes en una determinada ciudad o barrio. Has recibido la bolsa de la persona anterior de la cadena? ¡Ahora es el momento de comprar! Toma los objetos que quieras y pon ropa en la bolsa que estén en buenas condiciones y listos para un nuevo propietario. Siéntate libre de compartir una foto de tu última adquisición en el canal de comunicación correspondiente. <strong>¡Es muy agradable ver dónde acaban esas prendas de vestir!</strong> Entonces llevas la bolsa a la siguiente persona de la lista. No todos los participantes se conocen entre sí, así que realmente aumenta el sentido de comunidad en el vecindario.</p>",
   "wantToJoin": "¿Quieres unirte?",
-  "callToAction": "<p>¡Eres bienvenido! <aFind>Echa un vistazo a nuestro mapa</aFind> y regístrate en una cadena cerca de ti. ¿Todavía no hay una cadena activa para que te unas en tu área? ¡Estamos encantados de ayudarte a configurarlo! Unirse es gratis y abierto a todos.</p>",
```

Comment:
See if you can change the translated pieces too.

Reply:
Can you clarify what you mean?